mobile/lib/widgets/common/app_bar_dialog/app_bar_dialog.dart

```diff
@@ -116,10 +115,8 @@ class ImmichAppBarDialog extends HookConsumerWidget {
                 ok: "app_bar_signout_dialog_ok",
                 onOk: () async {
                   await ref.read(authenticationProvider.notifier).logout();
-
                   ref.read(manualUploadProvider.notifier).cancelBackup();
                   ref.read(backupProvider.notifier).cancelBackup();
-                  ref.read(assetProvider.notifier).clearAllAsset();
```

Comment:
Contrary to Alex's message, I am dubious about changing this as part of these changes as this is existing behaviour. Feels like it could potentially introduce bugs and should be done seperately if we want to change that behaviour?